CorsixTH/Src/th_map.cpp

```diff
@@ -996,7 +996,8 @@ void level_map::draw(render_target* pCanvas, int iScreenX, int iScreenY,
     blocks->draw_sprite(
         pCanvas, iBlock & 0xFF,
         itrNode1.tile_x_position_on_screen() + iCanvasX - 32,
-        itrNode1.tile_y_position_on_screen() + iCanvasY - iH + 32, iBlock >> 8);
+        itrNode1.tile_y_position_on_screen() + iCanvasY - iH + 32,
+        iBlock >> 8 | thdf_nearest);
```

Comment:
I'd prefer some parentheses here (and at other places as well, as it was added several times)

Reply:
Absolutely, makes sense! Done.